### PR TITLE
Feat: Broker Auth + Additional Targets

### PR DESCRIPTION
**Pull request by YOU54F**
Hey @guiyomh 

Thanks for releasing this tool, I remember when you told us about it back in December, I came back to my mind and thought it would be good to get auth based support in place, it looks like you had most of the work done, just needed exposing in user flags.

I also added a load more targets as well, because rustup/cross is awesome :)

Happy to split out the changes.

There are a couple of things that will need removing prior to merging, I set the value manually in the workflow file, as it wasn't picking up the next version, and updated a load of the action tags as they were deprecated.

### CODE REVIEW

**Comment by guiyomh** [comment]
@YOU54F thank for your contribution.I'll review it today. Sorry for the tartive review.

**Review by guiyomh — APPROVED** [review]

**Review by guiyomh — CHANGES_REQUESTED** [review]
The changelog is generated by the pipeline, so you need to remove it from your PR to avoid conflicts.

Similarly, version bumping is done automatically by the pipeline. You can't predict what the version number will be, it depends on the merge order of the PRs. So leave it to the pipeline.

Otherwise, thanks for the contribution, it's great 🤩 .

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,46 @@
...
+### Ci
+
+- Fix cargo-bump with cargo-release and create more targets
+- Add tag
+- Cargo check:
+- Cargo release:
+- Cargo release:
+- Ring 16.20_alpha for aarch64-pc-windows-msvc + latest cross for android:
+- Default bash shell for windows runners:
+
```
Comment (guiyomh) [inline CHANGELOG.md]: Automatically generated by the CI. Please remote it from your PR.

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pact-graph-network"
-version = "0.7.2"
+version = "0.8.0"
```
Comment (guiyomh) [inline Cargo.toml]: Bumping the  version is done by the pipeline.  Please remove it from your PR.

**Comment by YOU54F** [comment]
No rush at all @guiyomh , thanks for taking a look 👍🏾 

Changes addressed now and merged main in.

thanks buddy

**Review by guiyomh — APPROVED** [review]

**Comment by guiyomh** [comment]
It's merged! thank for your contrib.

**Pull request merged**